ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -276,6 +276,11 @@ private SafeFuture<BlockImportResult> handleBlockAndBlobsSidecarImport(
                     failedPayloadExecutionSubscribers.deliver(
                         FailedPayloadExecutionSubscriber::onPayloadExecutionFailed, block);
                     break;
+                  case FAILED_BLOBS_AVAILABILITY_CHECK:
+                    // TODO: we cannot invalidate the block root since it passed the checks
```

Comment:
I think that by the logic, there should be something similar to `UNKNOWN_PARENT` block here, but as it's our `NOOP` logic, we should just pass it. So it's a bit weird stub

Reply:
Yes, good point. I'll mention the comment in the `FetchRecentBlocksService` issue description

Reply:
for the records, our NOOP will not trigger this anymore.